src/twisted/trial/test/test_warning.py

```diff
@@ -365,6 +365,24 @@ def foo():
         # Flush it
         self.assertEqual(len(self.flushWarnings([module.foo])), 1)
 
+    def test_pep626(self):
+        """
+        dis.findlinestarts is not guaranteed to be sorted ascending.
+        """
+
+        def foo(a=1, b=1):
+            if a:
+                if b:
+                    warnings.warn("oh no")
```

Comment:
Maybe add a comment here that this needs to be constructed in this way to trigger findlinestarts behaviour